app/src/main/res/layout/dialog_circuit_delete.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:padding="@dimen/small_element_padding">
+
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:padding="@dimen/small_element_padding"
+        android:text="Delete Circuit"
+        android:textSize="@dimen/xl_large_text"
+        android:textStyle="bold" />
+
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:padding="@dimen/small_element_padding"
+        android:text="Do you want to delete this circuit?"
```

Comment:
We should display the name of the circuit being deleted. Should be a simple fix, set the text when you're displaying the dialog